example_backend_profile/settings.py

```diff
@@ -9,11 +9,11 @@
     DEBUG=(bool, True),
     SECRET_KEY=(str, ""),
     ALLOWED_HOSTS=(list, []),
-    OIDC_AUDIENCE=(str, ""),
-    OIDC_API_SCOPE_PREFIX=(str, ""),
-    OIDC_API_AUTHORIZATION_FIELD=(str, ""),
+    OIDC_AUDIENCE=(list, []),
+    OIDC_API_SCOPE_PREFIX=(str, []),
```

Comment:
`OIDC_API_SCOPE_PREFIX` type is still `str`.

Reply:
Ah! Fixed.